tasks/install_python3_ubuntu.yml

```diff
@@ -12,25 +12,30 @@
     vars:
       packages:
         - "{{ 'python%s.%s' % (python3_version.major, python3_version.minor) }}"
+        - "{{ 'python%s.%s-venv' % (python3_version.major, python3_version.minor) }}"
         - "{{ 'libpython%s.%s-dev' % (python3_version.major, python3_version.minor) }}"
 
-  - name: '[PYTHON3] locating python3 binary'
-    shell: python3 -c "import sys; print('%d.%d\n%s' % (sys.version_info.major, sys.version_info.minor, sys.executable))"
+  - name: '[PYTHON3] locating python3 binaries'
+    shell: "{{ item.key }} -c \"{{ item.value }}\""
     register: python3_output_ubuntu
     ignore_errors: False
+    loop:
+      - key: python3
+        value: import sys; print('system.builder.command.python%d\n%s' % (sys.version_info.major, sys.executable))
```

Comment:
would it be possible to pass only the part that is specific to each of the keys instead ? (such as `python%d.%d`)

Reply:
Fixed as well!